src/content/release-notes.mdx

```diff
@@ -5,6 +5,37 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.37.0
+
+3 October 2025
```

Comment:
Let's update to October 2. We could even do the release tomorrow.
Once the docs are ready, we are ready to go :-)